__tests__/extend.test.js

```diff
@@ -61,10 +61,33 @@ describe('extend', () => {
       ).toThrow('include must be an array');
     });
 
-    it('should throw if a path in the includes array doesnt resolve', () => {
-      expect(
-        StyleDictionary.extend.bind(null, {include: ['foo']})
-      ).toThrow("Cannot find module 'foo'");
+    it('should not update properties if include glob paths dont resolve to anything', () => {
+      var StyleDictionaryExtended = StyleDictionary.extend({
+        include: ['foo']
+      });
+      expect(typeof StyleDictionaryExtended.properties.size).toBe('undefined');
+    });
+
+    it('should properly glob paths', () => {
+      var StyleDictionaryExtended = StyleDictionary.extend({
+        include: [__dirname + '/__properties/*.json']
+      });
+      expect(typeof StyleDictionaryExtended.properties.size.padding.tiny).toBe('object');
+    });
+
+    it('should build the properties object if an include is given', () => {
+      var StyleDictionaryExtended = StyleDictionary.extend({
+        "include": [__dirname + "/__properties/paddings.json"]
+      });
+      expect(StyleDictionaryExtended.properties).toEqual(helpers.fileToJSON(__dirname + "/__properties/paddings.json"));
+    });
+
+    it('should override existing properties include is given', () => {
```

Comment:
Not my native language but is this description correct?

Reply:
Good catch, looks like it is missing 'if' in between 'properties' and 'include'.